packages/core/core.ts

```diff
@@ -70,7 +67,6 @@ export function withBemMod<T, U extends IClassNameProps = {}>(
 
     function BemMod(props: T & K) {
       modNames = modNames || Object.keys(mod)
-      // TODO: For performance can rewrite `every` to `for (;;)`.
```

Comment:
This PR not about this, let's returns this comment

Reply:
agree, sorry